docs/how-to-guides/develop-a-tool/create-and-use-tool-package.md

```diff
@@ -144,8 +144,13 @@ Alternatively, you can test your tool package using the script below to ensure t
           ----identifier
           {'module': 'module_name', 'package': 'package_name', 'package_version': 'package_version', ...}
           """
-          for entry_point in pkg_resources.iter_entry_points(group="package_tools"):
-              list_tool_func = entry_point.resolve()
+          entry_points = importlib.metadata.entry_points()
+          if isinstance(entry_points, list):
```

Comment:
What's the difference between the if and else?

Reply:
In python3.10 and later, the entry_points() method returns a SelectableView of EntryPoint objects, which allows us to select entry points by group. In the previous versions, the entry_points() method returns a dictionary-like object, we can use group name directly as a key.